kanidm_proto/src/v1.rs

```diff
@@ -313,18 +314,21 @@ impl fmt::Display for AuthType {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Ord, PartialOrd, Eq, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum UiHint {
+    PosixAccount,
+}
+
 /// The currently authenticated user, and any required metadata for them
 /// to properly authorise them. This is similar in nature to oauth and the krb
-/// PAC/PAD structures. Currently we only use this internally, but we should
-/// consider making it "parseable" by the client so they can have per-session
-/// group/authorisation data.
+/// PAC/PAD structures. This information is transparent to clients and CAN
+/// be parsed by them!
 ///
 /// This structure and how it works will *very much* change over time from this
-/// point onward!
-///
-/// It's likely that this must have a relationship to the server's user structure
-/// and to the Entry so that filters or access controls can be applied.
-#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+/// point onward! This means on updates, that sessions will invalidate in many
+/// cases.
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
As discussed, we need to change expiry in UserAuthToken to be serialized as seconds since epoch.

[the time serde docs](https://docs.rs/time/latest/time/serde/index.html) indicate adding this should sort it:

```
 #[serde(with = "time::serde::timestamp")
 ```